SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/OwnershipLockingPolicy/OwnershipLockingPolicyTest.cpp

```diff
@@ -313,7 +298,7 @@ void SpawnABCDHierarchy(FAutomationTestBase* Test, TSharedPtr<TestData> Data)
 	//     /
 	//    C
 
-	ADD_LATENT_AUTOMATION_COMMAND(FSpawnActor(Data, "A"));
+	ADD_LATENT_AUTOMATION_COMMAND(FSpawnActor(Data, "A"), ROLE_Authority);
```

Comment:
Should this be two lines, one spawning and one setting auth?

Reply:
nah this was an accident actually
```suggestion
	ADD_LATENT_AUTOMATION_COMMAND(FSpawnActor(Data, "A"));
```